pyspeckit/spectrum/fitters.py

```diff
@@ -656,7 +623,7 @@ def history_fitpars(self):
             for par in self.parinfo:
                 history.write_history(self.Spectrum.header, str(par))
                 
-    def peakbgfit(self, usemoments=True, annotate=None, vheight=True, height=0,
+    def peakbgfit(self, usemoments=True, annotate=None, vheight=False, height=0,
```

Comment:
I’m having some trouble with the vheight=True. 
In order to pass the assertion:
```
assert np.abs(measured_fwhm2-fitted_fwhm) < 200
```
I had to change test_fits.py:19 from sp.specfit() to sp.specfit.peakbgfit() (since specfit defaults to multifit) and having vheight=True gives me wrong results. I imagine this wasn’t possible before since peakbgfit was called from specfit.__call__ but I’m not sure.